shard-operator-parent/shard-operator-core/src/test/java/com/redhat/service/smartevents/shard/operator/core/resources/CustomResourceStatusTest.java

```diff
@@ -52,25 +44,6 @@ public void testMarkConditionFailure() {
         });
     }
 
-    @ParameterizedTest
-    @MethodSource("inferManagedResourceStatusParams")
```

Comment:
You can also delete the "method source" [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/c73169cc6cb7b44e0405ff1cc70347a53cce9217/shard-operator-parent/shard-operator-core/src/test/java/com/redhat/service/smartevents/shard/operator/core/resources/CustomResourceStatusTest.java#L47-L53).

Reply:
thanks for pointed it out. I removed it.